invenio_app_ils/closures/views.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025-2025 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio App ILS Closure views."""
+
+from flask import Blueprint, abort
+from flask_login import current_user
+from datetime import date
+from invenio_records_rest.views import pass_record
+from invenio_rest import ContentNegotiatedMethodView
+from invenio_app_ils.locations.api import LOCATION_PID_TYPE
+from invenio_app_ils.closures.serializers import closure_periods_response
+from invenio_app_ils.closures.api import get_closure_periods
+from invenio_app_ils.permissions import backoffice_permission
+from invenio_app_ils.records.permissions import RecordPermission
+
+
+def create_closures_blueprint(app):
+    """Create location closed periods blueprint."""
+    blueprint = Blueprint("invenio_app_ils_closures", __name__, url_prefix="")
+    endpoints = app.config.get("RECORDS_REST_ENDPOINTS", [])
```

Comment:
Given that if the config is wrong, everything will be broken, I would avoid `.get()` and fail early.
```suggestion
    endpoints = app.config["RECORDS_REST_ENDPOINTS"]
```

I guess it is appearing in multiple places (let's ignore those for the moment)